registry/config_windows.go

```diff
@@ -6,12 +6,20 @@ import (
 	"strings"
 )
 
-// DefaultV2Registry is the URI of the default (official) v2 registry.
-// This is the windows-specific endpoint.
-//
-// Currently it is a TEMPORARY link that allows Microsoft to continue
-// development of Docker Engine for Windows.
-const DefaultV2Registry = "https://ms-tp3.registry-1.docker.io"
+const (
+	// DefaultV1Registry is the URI of the default v1 registry
+	DefaultV1Registry = "https://registry-win-tp3.docker.io"
+
+	// DefaultV2Registry is the URI of the default (official) v2 registry.
+	// This is the windows-specific endpoint.
+	//
+	// Currently it is a TEMPORARY link that allows Microsoft to continue
+	// development of Docker Engine for Windows.
+	DefaultV2Registry = "https://registry-win-tp3.docker.io"
+
+	// SearchURL is the url used by the search function
+	SearchURL = DefaultV2Registry
```

Comment:
The `SearchRepositories` function is actually fixed to use `v1` here so doesn't matter what you add. Even if you add `v2` it will still search under `v1`.